packages/nimble-components/src/table/models/tests/table-validator.spec.ts

```diff
@@ -238,6 +244,82 @@ describe('TableValidator', () => {
         });
     });
 
+    describe('column config validation', () => {
```

Comment:
Can you add tests for
1. a column that starts out invalid but becomes valid 
2. a column that starts out valid but becomes invalid

It would be good to know that the table will observe these changes and update its own state, not just recognize columns that are always invalid / valid.

Reply:
Added tests that cover this.